<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Chapter 4 Chapter 3. Interactive Visualizations in Tableau | Statistical Analysis of the Human Gut Microbiome</title>
  <meta name="description" content="Chapter 4 Chapter 3. Interactive Visualizations in Tableau | Statistical Analysis of the Human Gut Microbiome">
  <meta name="generator" content="bookdown  and GitBook 2.6.7">

  <meta property="og:title" content="Chapter 4 Chapter 3. Interactive Visualizations in Tableau | Statistical Analysis of the Human Gut Microbiome" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 4 Chapter 3. Interactive Visualizations in Tableau | Statistical Analysis of the Human Gut Microbiome" />
  
  
  

<meta name="author" content="Vivek Sriram">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="3-chapter-2-post-processing-and-filtration-of-the-sequence-table.html">
<link rel="next" href="5-chapter-4-linear-regression-on-summary-statistics.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />









<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="2-chapter-1-pre-processing-of-microbiome-sequencing-data.html"><a href="2-chapter-1-pre-processing-of-microbiome-sequencing-data.html"><i class="fa fa-check"></i><b>2</b> Chapter 1: Pre-processing of Microbiome Sequencing Data</a></li>
<li class="chapter" data-level="3" data-path="3-chapter-2-post-processing-and-filtration-of-the-sequence-table.html"><a href="3-chapter-2-post-processing-and-filtration-of-the-sequence-table.html"><i class="fa fa-check"></i><b>3</b> Chapter 2. Post-processing and Filtration of the Sequence Table</a></li>
<li class="chapter" data-level="4" data-path="4-chapter-3-interactive-visualizations-in-tableau.html"><a href="4-chapter-3-interactive-visualizations-in-tableau.html"><i class="fa fa-check"></i><b>4</b> Chapter 3. Interactive Visualizations in Tableau</a></li>
<li class="chapter" data-level="5" data-path="5-chapter-4-linear-regression-on-summary-statistics.html"><a href="5-chapter-4-linear-regression-on-summary-statistics.html"><i class="fa fa-check"></i><b>5</b> Chapter 4. Linear Regression on Summary Statistics</a></li>
<li class="chapter" data-level="6" data-path="6-chapter-5-phylogenetic-tree-decomposition.html"><a href="6-chapter-5-phylogenetic-tree-decomposition.html"><i class="fa fa-check"></i><b>6</b> Chapter 5. Phylogenetic Tree Decomposition</a></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-the-first-appendix.html"><a href="A-the-first-appendix.html"><i class="fa fa-check"></i><b>A</b> The First Appendix</a></li>
<li class="chapter" data-level="B" data-path="B-the-second-appendix-for-fun.html"><a href="B-the-second-appendix-for-fun.html"><i class="fa fa-check"></i><b>B</b> The Second Appendix, for Fun</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Statistical Analysis of the Human Gut Microbiome</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="chapter-3.-interactive-visualizations-in-tableau" class="section level1">
<h1><span class="header-section-number">Chapter 4</span> Chapter 3. Interactive Visualizations in Tableau</h1>
<p>Tableau is a user-friendly application helpful in the design of interactive visualizations. We used this application to visualize the our data over time in our ordination plots and determine if there were any noticeable associations between variables in our metadata and the dynamics of our samples.</p>
<p>In each Tableau sheet created, hovering over a single point will cause that sample, as well as all other samples corresponding to the same patient, to be highlighted. This behavior allows us to specifically visualize how our samplesâ€™ distributions change on an individual basis.</p>
<p>Figures 8, 9, and 10 include some examples of the plots designed in Tableau.</p>
<p>Figure 8 depicts an ordination plot, colored by Batch number, and connected by patient ID. The thickness of lines connecting points to one another increases as time progresses. Hovering over a point will also highlight other sample points and the path representative of the selected patient.</p>
<p>
<div class="figure">
<img src="/Users/viveksriram/Desktop/ThesisFigures/figure8.png" alt="Ordination Plot colored by Batch Number and connected by Patient ID" />
<p class="caption">Ordination Plot colored by Batch Number and connected by Patient ID</p>
</div>
</p>
<p>Figure 9 shows plots of the maximum distance travelled by different patients, separated by whether or not their first sample date occurred before or after their transplant. It appears roughly that patients with first time points occurring before transplant date may have potentially higher maximum distances.</p>
<p>
<div class="figure">
<img src="/Users/viveksriram/Desktop/ThesisFigures/figure9.png" alt="Plots of Maximum Distance traveled by Patient Microbiome" />
<p class="caption">Plots of Maximum Distance traveled by Patient Microbiome</p>
</div>
</p>
<p>Figure 10 depicts two ordination plots colored by presence of acute graft versus host disease. The top plot depicts agvhd values taken within the twelve weeks prior to transplants, and the second plot depicts agvhd values taken within the twelve weeks after. Both plots can be manipulated to specify the desired time frame. This kind of visualization allows us to more directly compare the distributions of our covariates at different times.</p>
<p>
<div class="figure">
<img src="/Users/viveksriram/Desktop/ThesisFigures/figure10.png" alt="Ordination Plots colored by AGVHD, separated by Time" />
<p class="caption">Ordination Plots colored by AGVHD, separated by Time</p>
</div>
</p>
<p>With all of these plots, we were better able to interact with our data and the metrics that we had devised. However, while these visualizations were informative, we were still unable to specifically quantify potential associations between our various covariates and the dynamics of the gut microbiome.</p>

</div>
            </section>

          </div>
        </div>
      </div>
<a href="3-chapter-2-post-processing-and-filtration-of-the-sequence-table.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="5-chapter-4-linear-regression-on-summary-statistics.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
