<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Chapter 4 Linear Regression on Summary Statistics | Statistical Analysis of the Human Gut Microbiome</title>
  <meta name="description" content="Chapter 4 Linear Regression on Summary Statistics | Statistical Analysis of the Human Gut Microbiome">
  <meta name="generator" content="bookdown  and GitBook 2.6.7">

  <meta property="og:title" content="Chapter 4 Linear Regression on Summary Statistics | Statistical Analysis of the Human Gut Microbiome" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 4 Linear Regression on Summary Statistics | Statistical Analysis of the Human Gut Microbiome" />
  
  
  

<meta name="author" content="Vivek Sriram">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="3-interactive-visualizations-in-tableau.html">
<link rel="next" href="5-phylogenetic-tree-decomposition.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="1" data-path="1-pre-processing-of-microbiome-sequencing-data.html"><a href="1-pre-processing-of-microbiome-sequencing-data.html"><i class="fa fa-check"></i><b>1</b> Pre-processing of Microbiome Sequencing Data</a></li>
<li class="chapter" data-level="2" data-path="2-post-processing-and-filtration-of-the-sequence-table.html"><a href="2-post-processing-and-filtration-of-the-sequence-table.html"><i class="fa fa-check"></i><b>2</b> Post-processing and Filtration of the Sequence Table</a></li>
<li class="chapter" data-level="3" data-path="3-interactive-visualizations-in-tableau.html"><a href="3-interactive-visualizations-in-tableau.html"><i class="fa fa-check"></i><b>3</b> Interactive Visualizations in Tableau</a></li>
<li class="chapter" data-level="4" data-path="4-linear-regression-on-summary-statistics.html"><a href="4-linear-regression-on-summary-statistics.html"><i class="fa fa-check"></i><b>4</b> Linear Regression on Summary Statistics</a></li>
<li class="chapter" data-level="5" data-path="5-phylogenetic-tree-decomposition.html"><a href="5-phylogenetic-tree-decomposition.html"><i class="fa fa-check"></i><b>5</b> Phylogenetic Tree Decomposition</a></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Statistical Analysis of the Human Gut Microbiome</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="linear-regression-on-summary-statistics" class="section level1">
<h1><span class="header-section-number">Chapter 4</span> Linear Regression on Summary Statistics</h1>
<p>With the processing and filtration of our data, as well as its visualization in Tableau, we turned to basic linear regressions as well as non-parametric statistical tests to attempt to evaluate how our covariates may be linked to the dynamics of the microbiome.</p>
<p>We began with various non-parametric tests, because they avoid the distribution assumptions requisite in linear regressions. While these methods limit the amount of information we can consider within a single test, because we consider just one variable instead of several, they are still potentially useful in identifying traits related to our summary statistics.</p>
<p>Performing these tests for our maximum distance metric, we see that graft source appears to be significant. When we perform these same tests for our last distance metric, we see that none of our variables are significant (Tables 3 and 4).</p>
<p>
<table style="width:89%;">
<caption>Table 3. P-values from non-parametric statistical tests for maximum distance as outcome</caption>
<colgroup>
<col width="56%" />
<col width="31%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">Covariate</th>
<th align="center">P-value</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">Chronic Graft vs Host Disease</td>
<td align="center">0.35</td>
</tr>
<tr class="even">
<td align="center">Acute Graft vs Host Disease</td>
<td align="center">0.36</td>
</tr>
<tr class="odd">
<td align="center">Gender</td>
<td align="center">0.63</td>
</tr>
<tr class="even">
<td align="center">Vital Status</td>
<td align="center">0.14</td>
</tr>
<tr class="odd">
<td align="center">initialNegative</td>
<td align="center">0.68</td>
</tr>
<tr class="even">
<td align="center">Batch</td>
<td align="center">0.48</td>
</tr>
<tr class="odd">
<td align="center">Race</td>
<td align="center">0.33</td>
</tr>
<tr class="even">
<td align="center">Hispanic</td>
<td align="center">0.15</td>
</tr>
<tr class="odd">
<td align="center">Diagnosis</td>
<td align="center">0.91</td>
</tr>
<tr class="even">
<td align="center">Transplant Type</td>
<td align="center">0.28</td>
</tr>
<tr class="odd">
<td align="center">Transplant Response</td>
<td align="center">0.70</td>
</tr>
<tr class="even">
<td align="center">Graft Source</td>
<td align="center"><span class="math inline">\(1.0*10^{-2}\)</span></td>
</tr>
<tr class="odd">
<td align="center">Care Environment</td>
<td align="center">0.60</td>
</tr>
</tbody>
</table>
</p>
<p>
<table style="width:86%;">
<caption>Table 4. P-values from non-parametric statistical tests for last distance as outcome</caption>
<colgroup>
<col width="54%" />
<col width="31%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">Covariate</th>
<th align="center">P-value</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">Chronic Graft vs Host Disease</td>
<td align="center">0.51</td>
</tr>
<tr class="even">
<td align="center">Acute Graft vs Host Disease</td>
<td align="center">0.70</td>
</tr>
<tr class="odd">
<td align="center">Gender</td>
<td align="center">0.61</td>
</tr>
<tr class="even">
<td align="center">Vital Status</td>
<td align="center">0.17</td>
</tr>
<tr class="odd">
<td align="center">initialNegative</td>
<td align="center">0.43</td>
</tr>
<tr class="even">
<td align="center">Batch</td>
<td align="center">0.14</td>
</tr>
<tr class="odd">
<td align="center">Race</td>
<td align="center">0.43</td>
</tr>
<tr class="even">
<td align="center">Hispanic</td>
<td align="center">0.45</td>
</tr>
<tr class="odd">
<td align="center">Diagnosis</td>
<td align="center">0.77</td>
</tr>
<tr class="even">
<td align="center">Transplant Type</td>
<td align="center">0.78</td>
</tr>
<tr class="odd">
<td align="center">Transplant Response</td>
<td align="center">0.65</td>
</tr>
<tr class="even">
<td align="center">Graft Source</td>
<td align="center">0.15</td>
</tr>
<tr class="odd">
<td align="center">Care Environment</td>
<td align="center">0.78</td>
</tr>
</tbody>
</table>
</p>
<p>In addition to conducting our non-parametric tests for our summary statistics, we also performed linear regressions to attempt to quantify the significance of our covariates with respect to the dynamics of the microbiome.</p>
<p>Based upon exploratory data analysis, we kept all of our continuous variables untransformed. There were no extremely concerning trends in any of our categorical variables either, so we did not transform them. There was also no need to transform the outcome variable of maximum distance.</p>
<p>We used lattice plots to search for interaction effects across our categorical variables, as well as boxplots to search for connections between categorical and quantitative covariates. Identified interactions were included in our baseline model. Backwards selection was then performed using AIC as a scoring criterion.</p>
<p>For maximum distance as the outcome, our final model with the lowest AIC based on backward selection came out to be as follows:</p>
<p><span class="math inline">\(maxDistance_{i} = {\beta_{transplantAge}}transplantAge_{i} + {\beta_{AlloDLI}}I(transplantType_{i}=AlloDLI) + {\beta_{Auto}}I(transplantType_{i}=Auto) +{\beta_{Cord}}I(graftSource_{i}=Cord) + {\beta_{PBPC}}I(graftSource_{i}=PBPC) + {\beta_{Dead}}I(vitalStatus_{i}=Dead) + {\beta_{CGVHD}}I(CGVHD_{i}=1)+ {\beta_{anc500}}anc500_{i} + {\beta_{careEnv2}}I(careEnv_{i}=2) + {\beta_{careEnv3}}I(careEnv_{i}=3) + {\beta_{totalTime}}totalTime_{i} + {\beta_{vitalStatus/careEnv2}}I(vitalStatus/careEnv2_{i}=1) + {\beta_{vitalStatus/careEnv3}}I(vitalStatus/careEnv3_{i}=1)\)</span></p>
<p>This model had an AIC of -34.4, and it identified the following covariates as significant (Table 5):</p>
<p>
<table style="width:100%;">
<caption>Table 5. Significant P-values from linear regression with max distance as outcome</caption>
<colgroup>
<col width="35%" />
<col width="17%" />
<col width="32%" />
<col width="15%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">Covariate</th>
<th align="center">Estimate</th>
<th align="center">Confidence Interval</th>
<th align="center">P-value</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">Age at Transplant</td>
<td align="center"><span class="math inline">\(4.56*10^{-3}\)</span></td>
<td align="center"><span class="math inline">\((3.2*10^{-4}, 8.8*10^{-3})\)</span></td>
<td align="center"><span class="math inline">\(3.6*10^{-2}\)</span></td>
</tr>
<tr class="even">
<td align="center">Graft Source (Cord Blood)</td>
<td align="center"><span class="math inline">\(-3.91*10^{-1}\)</span></td>
<td align="center"><span class="math inline">\((-6.1*10^{-1}, -1.7*10^{-1})\)</span></td>
<td align="center"><span class="math inline">\(8.9*10^{-4}\)</span></td>
</tr>
<tr class="odd">
<td align="center">Vital Status (Dead)</td>
<td align="center"><span class="math inline">\(-9.14*10^{-2}\)</span></td>
<td align="center"><span class="math inline">\((-2.0*10^{-1}, 1.8*10^{-2})\)</span></td>
<td align="center"><span class="math inline">\(9.8*10^{-2}\)</span></td>
</tr>
<tr class="even">
<td align="center">Vital Status(Dead)/Care Env(3)</td>
<td align="center"><span class="math inline">\(2.86*10^{-1}\)</span></td>
<td align="center"><span class="math inline">\((-2.6*10^{-2}, 6.0*10^{-1})\)</span></td>
<td align="center"><span class="math inline">\(7.1*10^{-2}\)</span></td>
</tr>
</tbody>
</table>
</p>
<p>Based upon our model, we note that the “maximum distance” traveled in an ordination plot by a patient’s microbiome is expected to go up by roughly <span class="math inline">\(4.56*10^{-3}\)</span> units for each additional year of age at the time of transplant. In other words, it could be that the older a patient, the more variability in their microbiome.</p>
<p>We also see that a patient who ends up dead is expected to have a maximum distance that is roughly <span class="math inline">\(9.14*10^{-2}\)</span> units less than that of a patient who is still alive. A possible interpretation of this behavior is that patients who are still alive have microbiome compositions that vary significantly from their sick status pre-transplant.</p>
<p>Similarly, a patient who has a graft source of Cord Blood is expected to have a maximum distance that is roughly <span class="math inline">\(3.91*10^{-1}\)</span> units less than a patient who has a graft source of bone marrow. It may be the case that patients who receive cord blood stem cell donations have less disruption in their microbiomes than patients who receive bone marrow.</p>
<p>We note that our assumptions for equality of subpopulation standard deviations, as well as for our samples coming from a normally distributed population, are fulfilled by the residual plot and normal QQ-plot for our data.</p>
<p>We repeated the same backward selection process as above, but this time, we used last distance as our outcome variable. Based on backward selection according to AIC, we came up with the following model:</p>
<p><span class="math inline">\(lastDistance_{i} = {\beta_{Black}}I(race_{i}=Black) +{\beta_{MoreThanOne}}I(race_{i}=MoreThanOne) + {\beta_{Asian}}I(race_{i}=Asian) + {\beta_{hispanicUnk}}I(hispanic_{i}=Unk) + {\beta_{hispanicYes}}I(hispanic_{i}=Yes)+ {\beta_{Cord}}I(graftSource_{i}=Cord) + {\beta_{PBPC}}I(graftSource_{i}=PBPC) + {\beta_{Dead}}I(vitalStatus_{i}=Dead) + {\beta_{CGVHD}}I(CGVHD_{i}=1) + {\beta_{totalTimeSpan}}totalTimeSpan_{i}\)</span></p>
<p>Our model had an AIC of -32.3, and identified the following variables as significant (Table 6):</p>
<p>
<table>
<caption>Table 6. Significant P-values from linear regression with last distance as outcome</caption>
<colgroup>
<col width="30%" />
<col width="19%" />
<col width="34%" />
<col width="15%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">Covariate</th>
<th align="center">Estimate</th>
<th align="center">Confidence Interval</th>
<th align="center">P-value</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">Race (Black)</td>
<td align="center"><span class="math inline">\(1.59*10^{-1}\)</span></td>
<td align="center"><span class="math inline">\((-2.7*10^{-2}, 3.4*10^{-1})\)</span></td>
<td align="center"><span class="math inline">\(9.3*10^{-2}\)</span></td>
</tr>
<tr class="even">
<td align="center">Graft Source (Cord Blood)</td>
<td align="center"><span class="math inline">\(-3.01*10^{-1}\)</span></td>
<td align="center"><span class="math inline">\((-5.1*10^{-1}, -9.3*10^{-2})\)</span></td>
<td align="center"><span class="math inline">\(5.4*10^{-3}\)</span></td>
</tr>
<tr class="odd">
<td align="center">Vital Status (Dead)</td>
<td align="center"><span class="math inline">\(-1.09*10^{-1}\)</span></td>
<td align="center"><span class="math inline">\((-2.1*10^{-1}, -8.2*10^{-3})\)</span></td>
<td align="center"><span class="math inline">\(3.5*10^{-2}\)</span></td>
</tr>
<tr class="even">
<td align="center">Total Time Span</td>
<td align="center"><span class="math inline">\(7.59*10^{-4}\)</span></td>
<td align="center"><span class="math inline">\((1.6*10^{-5}, 1.5*10^{-3})\)</span></td>
<td align="center"><span class="math inline">\(4.6*10^{-2}\)</span></td>
</tr>
</tbody>
</table>
</p>
<p>Interestingly, graft source of cord blood and vital status still appear to be relevant. In particular, we see that according to our model, a patient who is confirmed dead is expected to have a last distance metric that is roughly <span class="math inline">\(1.09*10^{-1}\)</span> units shorter than that of a patient who is confirmed alive.</p>
<p>Likewise, a patient with a graft source of cord blood is expected to have a last distance value that is roughly <span class="math inline">\(3.01*10^{-1}\)</span> units shorter than that of a patient with a graft source of bone marrow. It seems that both of these traits are correlated with less change in the long-term for patients’ microbiome compositions. We also note that while transplant age is no longer significant, a race of “black” appears to be compared to the baseline of “asian.”</p>
<p>A major concern we had after these analyses was that our summary metrics could potentially vary extensively based upon the addition and removal of samples from our dataset. To deal with this possible high sensitivity to data, we decided to make use of a more creative method for modeling the abundances of ASVs in our data.</p>

</div>
            </section>

          </div>
        </div>
      </div>
<a href="3-interactive-visualizations-in-tableau.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="5-phylogenetic-tree-decomposition.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
