<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Chapter 6 Phylogenetic Tree Decomposition | Statistical Analysis of the Human Gut Microbiome</title>
  <meta name="description" content="Chapter 6 Phylogenetic Tree Decomposition | Statistical Analysis of the Human Gut Microbiome">
  <meta name="generator" content="bookdown  and GitBook 2.6.7">

  <meta property="og:title" content="Chapter 6 Phylogenetic Tree Decomposition | Statistical Analysis of the Human Gut Microbiome" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 6 Phylogenetic Tree Decomposition | Statistical Analysis of the Human Gut Microbiome" />
  
  
  

<meta name="author" content="Vivek Sriram">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="5-linear-regression-on-summary-statistics.html">
<link rel="next" href="conclusion.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="2-pre-processing-of-microbiome-sequencing-data.html"><a href="2-pre-processing-of-microbiome-sequencing-data.html"><i class="fa fa-check"></i><b>2</b> Pre-processing of Microbiome Sequencing Data</a></li>
<li class="chapter" data-level="3" data-path="3-post-processing-and-filtration-of-the-sequence-table.html"><a href="3-post-processing-and-filtration-of-the-sequence-table.html"><i class="fa fa-check"></i><b>3</b> Post-processing and Filtration of the Sequence Table</a></li>
<li class="chapter" data-level="4" data-path="4-interactive-visualizations-in-tableau.html"><a href="4-interactive-visualizations-in-tableau.html"><i class="fa fa-check"></i><b>4</b> Interactive Visualizations in Tableau</a></li>
<li class="chapter" data-level="5" data-path="5-linear-regression-on-summary-statistics.html"><a href="5-linear-regression-on-summary-statistics.html"><i class="fa fa-check"></i><b>5</b> Linear Regression on Summary Statistics</a></li>
<li class="chapter" data-level="6" data-path="6-phylogenetic-tree-decomposition.html"><a href="6-phylogenetic-tree-decomposition.html"><i class="fa fa-check"></i><b>6</b> Phylogenetic Tree Decomposition</a></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Statistical Analysis of the Human Gut Microbiome</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="phylogenetic-tree-decomposition" class="section level1">
<h1><span class="header-section-number">Chapter 6</span> Phylogenetic Tree Decomposition</h1>
<p>There are several aspects of microbiome data that make statistical analysis difficult. Potential issues include high dimensionality with large numbers of OTUs, sparsity due to small OTU counts, and potential correlations among counts of different OTUs. These aspects can cause problems when attempting to perform inference on the abundances of taxonomic units. Furthermore, simply analyzing regression results for a single OTU at a time fails to take into account the dependencies between different bacterial populations in the gut.</p>
<p>To address these concerns, we applied a Phylogenetic Tree Decomposition to our microbiome data. This methodology replicates concepts introduced in PhyloScan (Tang 2018) and DTM (Wang 2017). Using a phylogenetic tree can summarize the evolutionary relationships amongst the OTUs, allowing us to have a better context of their functional relationships and enriching the overall model fitting process.</p>
<p>With our completed filtration of our samples, we constructed a phylogenetic tree to represent the relations between our samples. We used the DECIPHER package in R to first perform multiple-alignment on the sequences in our ASV sequence table (Wright 2016). We then used the R package “phangorn” to fit a UPGMA tree based upon our sequences (Schliep 2018).</p>
<p>Our full tree was a binary tree with a single root node. There were a total of 11048 leaf nodes, and 11047 internal nodes. Our filtered ASV table has a total of 462 rows, each corresponding to a distinct sample. Each column in our ASV table represents a leaf in the phylogenetic tree. With these initial abundances for a given sample, we propagated our way up through the phylogenetic tree, determining the counts going left and right at each of our 11047 internal node. This process was repeated 462 times, once per sample. Figure 11 provides a smaller example of the phylogenetic tree transformation process. The code to achieve this transformation was implemented in Python, and took roughly two weeks to finish running.</p>
<p>
<div class="figure">
<img src="/Users/viveksriram/Desktop/ThesisFigures/figure15.png" alt="An example of bottom-up propagation of abundance counts" />
<p class="caption">An example of bottom-up propagation of abundance counts</p>
</div>
</p>
<p>In addition to using calculating counts going left and right at each internal node, code was also written in Python to determine the taxonomic rankings of each internal node, based upon the assigned taxonomies from the RDP for each ASV column in the sequence table. Figure 12 provides a smaller example of this propagation process.</p>
<p>
<div class="figure">
<img src="/Users/viveksriram/Desktop/ThesisFigures/figure16.png" alt="An example of bottom-up propagation of Taxonomies" />
<p class="caption">An example of bottom-up propagation of Taxonomies</p>
</div>
</p>
<p>With calculated counts and taxonomies for each of the internal nodes of our phylogenetic tree, we then applied a logit mixed-effect binomial model to each one, using the “glmer” function of the R “lme4” package (Bates 2015). At each node, we had 370 observations after filtration for missing values. There were a total of 129 different patients in our dataset. We incorporated a random effect for patient ID, and a nested random effect within patientID for sampleDate. We also included fixed effects for Batch number, transplant Age, gender, ethnicity, diagnosis, transplant type, graft source, vital status, care environment, presence of acute graft v. host disease, presence of chronic graft v. host disease, ANC500 level, and “preOrPost,” telling us if the first sample was taken before or after the transplant date.</p>
<p>The general format of our mixed effects model is as follows:</p>
<p><span class="math inline">\(logit(P_{i}(A)) = X_{i}\beta^{(A)} + \gamma_{i}^{(A)} + \epsilon_{it}^{(A)}\)</span></p>
<p><span class="math inline">\(P_{i}(A)\)</span> is the probability of picking the left child of node A for sample <span class="math inline">\(i\)</span> at time <span class="math inline">\(t\)</span>.</p>
<p><span class="math inline">\(\gamma_{i}^{(A)} = N(0, \sigma_{\gamma}^{2})\)</span></p>
<p><span class="math inline">\(\epsilon_{i}^{(A)} = N(0, \sigma_{\epsilon}^{2})\)</span></p>
<p><span class="math inline">\(for \ i = 1,...,370\)</span></p>
<p>In most cases, the regression model would fail to converge and provide meaningful results. However, there were twenty-two different internal nodes for which our model finished running and identified different variables to be significant. There were also several instances where Batch effects were noticed, including four nodes where the only significant variable was Batch number. The eighteen nodes that identified significant variables other than Batch are detailed in the table below, providing their taxonomy, the taxonomies of their children, and the significant covariates at their node.</p>
<p>
<table>
<caption>Significant Variables from Regression on Internal Nodes</caption>
<colgroup>
<col width="25%" />
<col width="50%" />
<col width="24%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">Taxonomy</th>
<th align="center">Children’s Taxonomies</th>
<th align="center">Significant Variables</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">None/NA</td>
<td align="center">Kingdom/Woesearchaeota, None/NA</td>
<td align="center">Diagnosis(MDS, Other), Graft Source(Cord, PBPC), Care Environment(2), ANC500</td>
</tr>
<tr class="even">
<td align="center">Domain/Bacteria</td>
<td align="center">Domain/Bacteria, Domain/Bacteria</td>
<td align="center">Gender(M), Hispanic(Yes), Transplant Type(Auto), Care Environment(2), preOrPost(pre)</td>
</tr>
<tr class="odd">
<td align="center">Phylum/Firmicutes</td>
<td align="center">Class/Selenomonadales, Kingdom/Firmicutes</td>
<td align="center">Gender(M), Hispanic(Unk), Care Environment(2), preOrPost(pre)</td>
</tr>
<tr class="even">
<td align="center">Domain/Bacteria</td>
<td align="center">Species/Gemella, Domain/Bacteria</td>
<td align="center">Transplant Age, Gender(M), Hispanic(Unk), Diagnosis(MDS), Transplant Type(Auto), preOrPost(pre), CGVHD</td>
</tr>
<tr class="odd">
<td align="center">Order/Ruminococcaceae</td>
<td align="center">Order/Ruminococcaceae, Order/Ruminococcaceae</td>
<td align="center">Hispanic(Yes), Graft Source(BM/PBPC)</td>
</tr>
<tr class="even">
<td align="center">Domain/Bacteria</td>
<td align="center">Domain/Bacteria, Kingdom/Firmicutes</td>
<td align="center">Hispanic(Yes), AGVHD</td>
</tr>
<tr class="odd">
<td align="center">Kingdom/Firmicutes</td>
<td align="center">Kingdom/Firmicutes, Order/Erysipelotrichaceae</td>
<td align="center">Graft Source(Cord Blood), Care Environment(2), ANC500</td>
</tr>
<tr class="even">
<td align="center">Order/Erysipelotrichaceae</td>
<td align="center">Order/Erysipelotrichaceae, Order/Erysipelotrichaceae</td>
<td align="center">Care Environment(3)</td>
</tr>
<tr class="odd">
<td align="center">Class/Lactobacillales</td>
<td align="center">Class/Lactobacillales, Class/Lactobacillales</td>
<td align="center">Hispanic(Yes), preOrPost(pre)</td>
</tr>
<tr class="even">
<td align="center">Kingdom/Firmicutes</td>
<td align="center">Order/Erysipelotrichaceae, Kingdom/Firmicutes</td>
<td align="center">preOrPost(pre)</td>
</tr>
<tr class="odd">
<td align="center">Order/Lactobacillaceae</td>
<td align="center">Family/Lactobacillus, Order/Lactobacillaceae</td>
<td align="center">Transplant Type(Auto), ANC500</td>
</tr>
<tr class="even">
<td align="center">Family/Streptococcus</td>
<td align="center">Family/Streptococcus, Family/Streptococcus</td>
<td align="center">Diagnosis(Other), CGVHD</td>
</tr>
<tr class="odd">
<td align="center">Family/Streptococcus</td>
<td align="center">Family/Streptococcus, Family/Streptococcus</td>
<td align="center">Transplant Age, Gender(M), Diagnosis(MDS, Other), Graft Source(Cord Blood)</td>
</tr>
<tr class="even">
<td align="center">Family/Bacteroides</td>
<td align="center">Family/Bacteroides, Family/Bacteroides</td>
<td align="center">Transplant Age, Hispanic(Unknown), Transplant Type(Allo-DLI)</td>
</tr>
<tr class="odd">
<td align="center">Family/Streptococcus</td>
<td align="center">Family/Streptococcus, Family/Streptococcus</td>
<td align="center">Graft Source(BM/PBPC, PBPC), CGVHD</td>
</tr>
<tr class="even">
<td align="center">Family/Bifidobacterium</td>
<td align="center">Family/Bifidobacterium, Family/Bifidobacterium</td>
<td align="center">Gender(M), Hispanic(Yes), Transplant Type(Auto), Graft Source(Cord Blood, PBPC), AGVHD</td>
</tr>
<tr class="odd">
<td align="center">Family/Streptococcus</td>
<td align="center">Family/Streptococcus, Family/Streptococcus</td>
<td align="center">AGVHD</td>
</tr>
<tr class="even">
<td align="center">Family/Bacteroides</td>
<td align="center">Family/Bacteroides, Family/Bacteroides</td>
<td align="center">Diagnosis(Other), Transplant Type(Auto), Care Environment(2)</td>
</tr>
</tbody>
</table>
</p>
<p>Common variables identified across taxonomic rankings include ethnicity, graft source, care environment, diagnosis, transplant type, and presence of acute and chronic graft vs. host disease.</p>

</div>
            </section>

          </div>
        </div>
      </div>
<a href="5-linear-regression-on-summary-statistics.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="conclusion.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
