<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>References | Statistical Analysis of Gut Microbiome Data for Memorial Sloan Kettering Patients Recovering from Leukemia</title>
  <meta name="description" content="References | Statistical Analysis of Gut Microbiome Data for Memorial Sloan Kettering Patients Recovering from Leukemia">
  <meta name="generator" content="bookdown  and GitBook 2.6.7">

  <meta property="og:title" content="References | Statistical Analysis of Gut Microbiome Data for Memorial Sloan Kettering Patients Recovering from Leukemia" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="References | Statistical Analysis of Gut Microbiome Data for Memorial Sloan Kettering Patients Recovering from Leukemia" />
  
  
  

<meta name="author" content="Vivek Sriram">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="conclusion.html">

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="1" data-path="1-pre-processing-of-microbiome-sequencing-data.html"><a href="1-pre-processing-of-microbiome-sequencing-data.html"><i class="fa fa-check"></i><b>1</b> Pre-processing of Microbiome Sequencing Data</a></li>
<li class="chapter" data-level="2" data-path="2-post-processing-and-filtration-of-the-sequence-table.html"><a href="2-post-processing-and-filtration-of-the-sequence-table.html"><i class="fa fa-check"></i><b>2</b> Post-processing and Filtration of the Sequence Table</a></li>
<li class="chapter" data-level="3" data-path="3-interactive-visualizations-in-tableau.html"><a href="3-interactive-visualizations-in-tableau.html"><i class="fa fa-check"></i><b>3</b> Interactive Visualizations in Tableau</a></li>
<li class="chapter" data-level="4" data-path="4-linear-regression-on-summary-statistics.html"><a href="4-linear-regression-on-summary-statistics.html"><i class="fa fa-check"></i><b>4</b> Linear Regression on Summary Statistics</a></li>
<li class="chapter" data-level="5" data-path="5-phylogenetic-tree-decomposition.html"><a href="5-phylogenetic-tree-decomposition.html"><i class="fa fa-check"></i><b>5</b> Phylogenetic Tree Decomposition</a></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Statistical Analysis of Gut Microbiome Data for Memorial Sloan Kettering Patients Recovering from Leukemia</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<!--
This manually sets the header for this unnumbered chapter.
-->

<!--
To remove the indentation of the first entry.
-->

<!--
To create a hanging indent and spacing between entries.  These three lines may need to be removed for styles that don't require the hanging indent.
-->
<p>  </p>
<!--
This is just for testing with more citations for the bibliography at the end.  Add other entries into the list here if you'd like them to appear in the bibliography even if they weren't explicitly cited in the document.
-->

<div id="refs" class="references">
<div>
<p>Bates, D., Machler, M., Bolker, B., &amp; Walker, S. (2015). Fitting linear mixed-effects models using lme. <em>Journal of Statistical Software</em>, <em>67</em>(1), 1–48. Retrieved from <a href="http://dx.doi.org/10.18637/jss.v067.i01" class="uri">http://dx.doi.org/10.18637/jss.v067.i01</a></p>
</div>
<div>
<p>Callahan, B., McMurdie, P., Paul, J., &amp; Holmes, S. (2017). Exact sequence variants should replace operational taxonomic units in marker-gene data analysis. <em>ISME</em>, <em>11</em>, 2639. Retrieved from <a href="https://www.nature.com/articles/ismej2017119  " class="uri">https://www.nature.com/articles/ismej2017119  </a></p>
</div>
<div>
<p>Callahan, B., McMurdie, P., Rosen, M., Han, A., Johnson, A., &amp; Holmes, S. (2016a). DADA2: High-resolution sample inference from illumina amplicon data. <em>Nat Methods</em>, <em>13</em>, 581–583. Retrieved from <a href="https://www.nature.com/articles/nmeth.3869 " class="uri">https://www.nature.com/articles/nmeth.3869 </a></p>
</div>
<div>
<p>Cole, J., Wang, Q., Fish, J., Chai, B., McGarrell, D., &amp; al., Y. S. et. (2014). Ribosomal database project: Data and tools for high throughput rRNA analysis. <em>Nucleic Acids Research</em>, <em>42</em>(D1), D633–D642. Retrieved from <a href="https://doi.org/10.1093/nar/gkt1244" class="uri">https://doi.org/10.1093/nar/gkt1244</a></p>
</div>
<div>
<p>Harrison, P., Alako, B., Amid, C., Cerdeno-Tarraga, A., Cleland, I., &amp; al., S. H. et. (2019). The european nucleotide archive in 2018. <em>Nucleic Acids Research</em>, <em>47</em>(D1), D84–D88. Retrieved from <a href="https://doi.org/10.1093/nar/gky1078" class="uri">https://doi.org/10.1093/nar/gky1078</a></p>
</div>
<div>
<p>Lozupone, C., Li, M., Campbell, T., Flores, S., Linderman, D., &amp; al., M. G. et. (2013). Alterations in the gut microbiota associated with hiv-1 infection. <em>Cell Host Microbe</em>, <em>14</em>(3), 329–339. Retrieved from <a href="https://www.sciencedirect.com/science/article/pii/S1931312813002886?via%3Dihub">https://www.sciencedirect.com/science/article/pii/S1931312813002886?via%3Dihub</a></p>
</div>
<div>
<p>McMurdie, P., &amp; Holmes, S. (2013). Phyloseq: An r package for reproducible interactive analysis and graphics of microbiome census data. <em>PLOS ONE</em>, <em>8</em>(4), e61217. Retrieved from <a href="https://doi.org/10.1371/journal.pone.0061217" class="uri">https://doi.org/10.1371/journal.pone.0061217</a></p>
</div>
<div>
<p>NIH. (2009). The nih human microbiome project. <em>Genome Research</em>, <em>19</em>(12), 2317–23. Retrieved from <a href="https://www.nih.gov/news-events/news-releases/nih-human-microbiome-project-defines-normal-bacterial-makeup-body" class="uri">https://www.nih.gov/news-events/news-releases/nih-human-microbiome-project-defines-normal-bacterial-makeup-body</a></p>
</div>
<div>
<p>Schliep, K. (2016). Estimating phylogenetic trees with phangorn. <em>Bioinformatics</em>, <em>27</em>, 592–593. Retrieved from <a href="https://cran.r-project.org/web/packages/phangorn/vignettes/Trees.pdf" class="uri">https://cran.r-project.org/web/packages/phangorn/vignettes/Trees.pdf</a></p>
</div>
<div>
<p>Tang, Y., Ma, L., &amp; Nicolae, D. (2018). A phylogenetic scan test on a dirichlet-tree multinomial model for microbiome data. <em>Ann. Appl. Stat</em>, <em>12</em>(1), 1–26. Retrieved from <a href="https://doi.org/10.1214/17-AOAS1086" class="uri">https://doi.org/10.1214/17-AOAS1086</a></p>
</div>
<div>
<p>Wang, T., &amp; Zhao, H. (2017). A dirichlet-tree multinomial regression model for associating dietary nutrients with gut microorganisms. <em>Biometrics</em>, <em>73</em>(3), 792–801. Retrieved from <a href="https://www.ncbi.nlm.nih.gov/pubmed/28112797" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/28112797</a></p>
</div>
<div>
<p>Wright, E. (2016). Using decipher v2.0 to analyze big biological sequence data in r. <em>The R Journal</em>, <em>8</em>(1), 352–359. Retrieved from <a href="https://bioconductor.org/packages/release/bioc/html/DECIPHER.html" class="uri">https://bioconductor.org/packages/release/bioc/html/DECIPHER.html</a></p>
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="conclusion.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
